lib/flow.js

```diff
@@ -2,8 +2,19 @@
 
 module.exports = (api) => {
 
+  api.metasync.flow = (
```

Comment:
But it is used by `parallel()` and `sequential()`. Do you plan to deprecate those too and make `flow()` not use them yet implement this functionality on its own or you're just talking about deprecating the part of the public interface, not the actual function, i.e., `composition()` will stay there, but it'll be a hidden implementation detail?